lib/takeout.rb

```diff
@@ -0,0 +1,61 @@
+class Takeout
+
+  def initialize
+    @menu = Menu.new
+    @order = Order.new
+    @message = Message.new
+    @delivery = Delivery.new
+  end
+
+  def build
+    item = ' str '
+    while item.downcase != 'end'
+      puts 'Enter item or type end to confirm'
+      item = gets.chomp!
+      return if item.downcase == 'end'
+      puts 'Enter number of items'
+      number = gets.chomp!
+      @order.add(item, number.to_f, @menu)
+    end
```

Comment:
I wonder if some of this could be passed to a `Printer` class or something. It seems very functional.